WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -163,9 +181,36 @@ private extension OrderListViewModel {
     }
 }
 
+// MARK: - Order Status
+
+private extension OrderListViewModel {
+    /// Setup: Status Results Controller
+    ///
+    func configureStatusResultsController() {
+        statusResultsController.predicate = NSPredicate(format: "siteID == %lld", siteID)
+        try? statusResultsController.performFetch()
```

Comment:
This is a low-hanging fruit and only if you have time. We can add error logging here. 

```suggestion
        do {
            try statusResultsController.performFetch()
        } catch {
            CrashLogging.logError(error)
        }
```

Reply:
Done! Thanks for the suggestion 👍